### DISCUSSION TITLE
Squares in the Kingdom Overview design

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [ihhub/fheroes2#6141] [General]
In the Town/Castles view in the Kingdom Overview, the creatures in fheroes2 are in square boxes. The original game doesn't have these boxes and in my mind it looks much simple and cleaner that way. I think the look for fheroes2 would look more polished if the boxes where removed. The boxes within boxes within boxes makes the Overview looks to messy, especially late game when you have several castles. 
 

OG game
![originalTownscreen](https://user-images.githubusercontent.com/93934125/201095582-4032e774-a01d-441a-b74a-76aee0974158.png)

Fheroes2
![fheroes2TownScreen](https://user-images.githubusercontent.com/93934125/201095989-139a45e2-80f5-4bed-b5ba-77ade1f5accb.png)

**Comment by LeHerosInconnu** [comment]
Hello @Mr-Bajs,

> In the Town/Castles view in the Kingdom Overview, the creatures in fheroes2 are in square boxes. The original game doesn't have these boxes and in my mind it looks much simple and cleaner that way. I think the look for fheroes2 would look more polished if the boxes where removed. The boxes within boxes within boxes makes the Overview looks to messy, especially late game when you have several castles.

This is indeed a problem. :)
I have made suggestions in this sense here: https://github.com/ihhub/fheroes2/issues/1844.

**Comment by oleg-derevenetz** [comment]
> The original game doesn't have these boxes and in my mind it looks much simple and cleaner that way.

The original game doesn't allow to manipulate troops from the Kingdom View - all you can do is just open the creature's info window. You cannot move it to a different slot or transfer between garrison and a guest hero. In fheroes2 you are able to do all this. So these slots are here for a reason (at least those in the central part of the row).

**Comment by AndersonLima12** [comment]
The original game doesn't allow you to manipulate Kingdom View troops - all you can do is just bring up a creature info window. You cannot move it to a different slot or transfer between garrison and a guest hero. With that you could make some specific adaptations for you.

**Reply by Novicek** [reply]
What about making the boxes appear only when you're actually selecting/dragging a creature then? It would still look clean when just looking over things, but functional when making any changes.
